migration/template/upgrade-apps.md

```diff
@@ -0,0 +1,60 @@
+# Upgrade v${old_version}.x to v${new_version}.x
+
+## Prerequisites
+
+- [ ] Notify the users (if any) before the upgrade starts;
+- [ ] Check if there are any pending changes to the environment;
+- [ ] Check the state of the environment, pods, nodes and backup jobs:
+
+    ```bash
+    ./bin/ck8s test sc|wc
+    kubectl get po -A
```

Comment:
To make it a bit easier to read
```suggestion
    kubectl get po -A | grep -vP "(Running|Completed)"
```

Reply:
There could be pods that are "Running" but not ready that wouldn't be caught when you do this

Reply:
not perfect, but would this work?
`kubectl get pods -A -o custom-columns=NAMESPACE:metadata.namespace,POD:metadata.name,READY-false:status.containerStatuses[*].ready | grep false`

Reply:
How does this work if there are multiple containers? Will the column `READY-false` just take one or does it have multiple values?

Reply:
Yea, then that would work